.changes/v2.14.0/418-bug-fixes.md

```diff
@@ -0,0 +1,2 @@
+* Fix Issue #728: `vm.UpdateInternalDisksAsync()` didn't send VM description and as result would delete it [GH-418]
```

Comment:
It's not clear what "it" refers to. Also, from the conversation in Issue #728, it doesn't seem that this fix addresses that problem.

Reply:
yes, it is not 100% as I had very limited information

Reply:
updated "it"